saw-script/src/SAWScript/Interpreter.hs

```diff
@@ -2425,14 +2425,14 @@ primTypes = Map.fromList
   , abstype "FunctionProfile" Experimental
   , abstype "FunctionSkeleton" Experimental
   , abstype "Ghost" Current
-  , abstype "JVMSetup" Current
+  , abstype' "JVMSetup" SS.kindStar Current
   , abstype "JVMValue" Current
   , abstype "JavaClass" Current
   , abstype "JavaType" Current
-  , abstype "LLVMSetup" Current
+  , abstype' "LLVMSetup" SS.kindStar Current
   , abstype "LLVMModule" Current
   , abstype "LLVMType" Current
-  , abstype "MIRSetup" Current
+  , abstype' "MIRSetup" SS.kindStar Current
```

Comment:
Similarly, using `kindStar` as the kinds of `{LLVM,JVM,MIR}Setup` is wrong, but we accept this at the moment pending a fix for https://github.com/GaloisInc/saw-script/issues/2764.